build.sbt

```diff
@@ -35,7 +35,7 @@ val prevArtifacts = Def.derive {
 
 def jsOpts = new Def.SettingList(Seq(
   scalaJSLinkerConfig ~= { _.withModuleKind(ModuleKind.CommonJSModule) },
-  crossScalaVersions := crossScalaVersions.value.filter(_.startsWith("2."))
+  crossScalaVersions := crossScalaVersions.value.filterNot(_ == "3.0.0-M1")
```

Comment:
Does `3.0.0-M2` have no issues with build for Scala.js?

Reply:
I just tired it and after applying what I said [here](https://github.com/Log4s/log4s/pull/64/files#r536001193), it still fails to compile for Scala.js for Scala 3.0.0-M2.

Reply:
I thought this compiled last night, but apparently not.  I don't particularly want to block a milestone on sorting this out.  Did you need it for your project, or is the JVM version sufficient?

Reply:
Yeah, I need it for my project and the JVM version is sufficient. It would be great, if it can be release. Thank you!